sdk/nodejs/cloudwatch/eventMetrics.ts

```diff
@@ -0,0 +1,94 @@
+// Copyright 2016-2018, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import * as pulumi from "@pulumi/pulumi";
+import { Metric, MetricChange } from "./metric";
+
+export type CloudwatchEventMetricName =
+    "DeadLetterInvocations" | "Invocations" | "FailedInvocations" | "TriggeredRules" |
+    "MatchedEvents" | "ThrottledRules";
```

Comment:
humorously (but for good purpose) CloudWatch publishes its own events on how it is being used.  So the aws.cloudwatch package is both whre we define things like the Metric/Alarm types, but it's also where you can get metrics for, and create alarms from, how your actual cloudwatch subsystem is being used.